ansible/main.yml

```diff
@@ -13,7 +13,9 @@
   hosts:
     - wakuv1-test
     - wakuv2-test
+    - wakuv2-staging
```

Comment:
Minor note, but I envisioned this to be a Status fleet (in infra-status) rather than a `wakuv2` fleet. On second thought, it may be a good idea to have our own fleet with static shard configuration as you've done here, but then I wouldn't call it "staging" - rather something like `wakuv2-static`.